src/foundry/foundry.js/clientDocuments/actor.d.ts

```diff
@@ -25,10 +27,267 @@ declare global {
+     * If viewing a full Actor entity, all Tokens which map to that actor will be targeted for initiative rolls.
+     * If viewing a synthetic Token actor, only that particular Token will be targeted for an initiative roll.
+     *
+     * @param options - Configuration for how initiative for this Actor is rolled.
+     * @returns A promise which resolves to the Combat entity once rolls are complete.
+     */
+    rollInitiative(options?: Actor.RollInitiativeOptions): Promise<void>;
+
+    /** {@inheritdoc} */
+    getEmbeddedCollection(embeddedName: string): Collection<foundry.abstract.Document<any, this>>;
+
+    /**
+     * {@inheritdoc}
+     * @override
+     */
+    _preCreate(
+      data: DeepPartial<foundry.documents.BaseActor>['data']['_source'],
+      options: DocumentModificationOptions,
+      user: BaseUser
+    ): Promise<void>;
+
+    /**
+     * {@inheritdoc}
+     * @override
+     */
+    _onUpdate(
+      data: DeepPartial<foundry.documents.BaseActor>['data']['_source'],
+      options: DocumentModificationOptions,
+      user: string
+    ): void;
+
+    /**
+     * {@inheritdoc}
+     * @override
+     */
+    _onCreateEmbeddedDocuments(
+      embeddedName: string,
+      documents: foundry.abstract.Document<any, any>[],
+      result: Record<string, unknown>[],
+      options: DocumentModificationOptions,
+      userId: string
+    ): void;
+
+    /**
+     * {@inheritdoc}
+     * @override
+     */
+    _onUpdateEmbeddedDocuments(
+      embeddedName: string,
+      documents: foundry.abstract.Document<any, any>[],
+      result: Record<string, unknown>[],
+      options: DocumentModificationOptions,
+      userId: string
+    ): void;
+
+    /**
+     * {@inheritdoc}
+     * @override
+     */
+    _onDeleteEmbeddedDocuments(
+      embeddedName: string,
+      documents: foundry.abstract.Document<any, any>[],
+      result: string[],
+      options: DocumentModificationContext,
+      userId: string
+    ): void;
+
+    /**
+     * Refresh the display of active Tokens for this Actor if ActiveEffects were changed
+     */
+    protected _refreshTokens(): void;
+
+    /**
+     * You are referencing Actor#_data which has been deprecated in favor of Actor#data#_source. Support for this reference will be removed in 0.9.0
+     * @deprecated since 0.8.0
+     */
+    get _data(): this['data']['_source'];
+
+    /**
+     * You are referencing Actor#getOwnedItem(itemId) which is deprecated in favor of Actor#items#get(itemId). Support will be removed in 0.9.0
+     * @deprecated since 0.8.0
+     */
+    getOwnedItem(itemId: string): InstanceType<ConfiguredDocumentClass<typeof BaseItem>>;
+
+    /**
+     * You are referencing Actor#createOwnedItem which is deprecated in favor of Item.create or Actor#createEmbeddedDocuments. Support will be removed in 0.9.0
+     * @deprecated since 0.8.0
+     */
+    createOwnedItem(
+      itemData: Parameters<this['createEmbeddedDocuments']>[1],
```

Comment:
This does not accurately reflect the type, a single value is also allowed, not only an array.